README.md

```diff
@@ -220,20 +233,19 @@ targetManager:
 
 #### Loader
 
-The loader component of the `sparrow` will load the [Runtime](#runtime) configuration dynamically.
+The loader component of the `sparrow` will load the [checks](#checks) configuration dynamically during runtime.
 
 The loader can be selected by specifying the `loaderType` configuration parameter.
 
-The default loader is an `http` loader that is able to get the runtime configuration from a remote endpoint.
+The default loader is a `http` loader that is able to get the checks configuration from a remote endpoint during runtime.
```

Comment:
An was correct, because http is pronounced eyyyhttp, so it's like it starts with a vowel. Or just use "the default type of the loader is `http`, which is able..."

Reply:
Also, "the checks' configuration"